### DISCUSSION TITLE
Would it be possible make the left bar extendable so we can see the full name of the pipelines?

### DISCUSSION THREAD

**Discussion by scharbo1** [concourse/concourse#7458] [Ideas]
![image](https://user-images.githubusercontent.com/89469973/130837083-9287f642-4fcc-4a4e-b55f-bc6133f6ab9c.png)

**Comment by aoldershaw** [comment]
The sidebar is expandable as of 6.4.0: https://github.com/concourse/concourse/releases/tag/v6.4.0#5782
